tests/src/test/java/alluxio/client/RemoteBlockInStreamIntegrationTest.java

```diff
@@ -248,8 +249,8 @@ public void readTest4() throws Exception {
       long blockId = mFileSystem.getStatus(uri).getBlockIds().get(0);
       BlockInfo info = AlluxioBlockStore.get().getInfo(blockId);
       WorkerNetAddress workerAddr = info.getLocations().get(0).getWorkerAddress();
-      RemoteBlockInStream is =
-          new RemoteBlockInStream(info.getBlockId(), info.getLength(), workerAddr);
+      RemoteBlockInStream is = new RemoteBlockInStream(info.getBlockId(), info.getLength(),
+          workerAddr, BlockStoreContext.INSTANCE);
```

Comment:
Which singletons are safe to use in tests? Or should singletons always be used in integration tests?